Header header

string chain_name

sensor_msgs/JointState source_state
sensor_msgs/JointState target_state

float64 cost
float64 ee_weight